### PR TITLE
Add cheat sheet tags for dynamic labels/types

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
(Also fixes 2 minor errors)

Corresponding Cheat sheet PR:
https://github.com/neo4j/docs-cheat-sheet/pull/202

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1117.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/create)
- [LOAD CSV](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/load-csv)
- [MATCH](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/match)
- [MERGE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/merge)

File: modules/ROOT/pages/clauses/create.adoc [hunk 1]
```diff
@@ -235,6 +235,7 @@ This is because a relationship can only have exactly one type.
 ----
 
 .Create nodes and relationships using dynamic node labels and relationship types
+// tag::dynamic_create[]
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/clauses/create.adoc]: a note about naming scheme -
for example, `functions_aggregating_avg` is a label for the avg function
i tried to make it the cypher manual section (functions), then the asciidoc file (or functions group, "aggregating") and then the function in that page

we probably do not have to be super religious with schemes, but i thought i'd bring it up :)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1117.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/create)
- [LOAD CSV](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/load-csv)
- [MATCH](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/match)
- [MERGE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/merge)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1117.surge.sh

Updated pages:
- [CREATE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/create)
- [LOAD CSV](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/load-csv)
- [MATCH](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/match)
- [MERGE](https://neo4j-docs-cypher-1117.surge.sh/cypher-manual/5/clauses/merge)

**Review by rsill-neo4j — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**